modules/account-lib/src/coin/hbar/transactionBuilder.ts

```diff
@@ -61,6 +80,43 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
   // endregion
 
   // region Common builder methods
+  /**
+   *  Set the memo
+   *
+   * @param {string} memo A hedera memo
+   * @returns {TransactionBuilder} This transaction builder
+   */
+  memo(memo: string): this {
+    this._memo = memo;
```

Comment:
Add some validations for the memo. Check valid characters and length

Reply:
That sounds good, we plan to use the hashgraph sdk validations as a base and then check if we need more validations.